LICENSE

```diff
@@ -1,6 +1,6 @@
 BSD 3-Clause License
 
-Copyright (c) 2017-2018 P. Angerer, F. Alexander Wolf, Theis Lab
+Copyright (c) 2025, scverse
```

Comment:
Wouldn't you just retain the previous copyright and add scverse additionally?

Reply:
That would make sense!